RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -14,6 +19,12 @@ MeasurementTrackerEventProducer::MeasurementTrackerEventProducer(const edm::Para
     std::vector<edm::InputTag> inactivePixelDetectorTags(iConfig.getParameter<std::vector<edm::InputTag> >("inactivePixelDetectorLabels"));
     for (auto &t : inactivePixelDetectorTags) theInactivePixelDetectorLabels.push_back(consumes<DetIdCollection>(t));
 
+    if ( iConfig.exists("badPixelFEDChannelCollectionLabels") ) {
+      std::vector<edm::InputTag> badPixelFEDChannelCollectionTags(iConfig.getParameter<std::vector<edm::InputTag> >("badPixelFEDChannelCollectionLabels"));
+      for (auto &t : badPixelFEDChannelCollectionTags) theBadPixelFEDChannelsLabels.push_back(consumes<PixelFEDChannelCollection>(t));
+      pixelCablingMapLabel_ = iConfig.getParameter<std::string> ("pixelCablingMapLabel");
```

Comment:
for this review, I'm assuming that this ```if (iConfig.exists``` is addressed in the follow up PR (same for a few other comments in the RecoTracker, unless noted otherwise).